src/pad/src/RDLRouter.cpp

```diff
@@ -154,16 +154,16 @@ RDLRoute::RDLRoute(odb::dbITerm* source,
   std::stable_sort(terminals_.begin(),
                    terminals_.end(),
                    [&iterm_center](odb::dbITerm* lhs, odb::dbITerm* rhs) {
-                     const bool lhs_cover = RDLRouter::isCoverTerm(lhs);
-                     const bool rhs_cover = RDLRouter::isCoverTerm(rhs);
+                     const int lhs_cover = RDLRouter::isCoverTerm(lhs) ? 1 : 0;
+                     const int rhs_cover = RDLRouter::isCoverTerm(rhs) ? 1 : 0;
```

Comment:
why convert from bool to int?

Reply:
It wasn't 100% clear to me how the bool comparison would, it wasn't clear from the cpp docs if true or false was smaller

Reply:
std::cout << (true < false);
    std::cout << (false < true);

01

Reply:
thanks for clarifying